wd_core.au3

```diff
@@ -150,7 +151,8 @@ Global Const $aWD_ERROR_DESC[$_WD_ERROR_COUNTER] = [ _
 		"File issue", _
 		"Browser or feature not supported", _
 		"Capability or value already defined", _
-		"Javascript Exception" _
+		"Javascript Exception", _
+		"Failed to decode request as JSON" _
```

Comment:
maybe change this to `Failed to decode JSON` ?

Reply:
I would suggest something more generic, such as `JSON Exception`